README.md

```diff
@@ -382,7 +382,6 @@ The `msg` is the same as the Message type. `opts` includes the `opts.colors` pas
   - `opts.convo_store` Implementation of ConversationStore, defaults to memory
   - `opts.context` `Function (req, res, next)` HTTP Middleware function to enrich incoming request with context
   - `opts.log` defaults to `true`, `false` to disable logging
-  - `opts.logger` Implementation of a logger, defaults to built-in Slapp command line logger.
```

Comment:
This probably just needs to be added to the docs since it was re-generated and had been manually added?

Reply:
I went ahead and fixed it